composer.json

```diff
@@ -18,24 +18,44 @@
 	},
 	"autoload" : {
 		"psr-4" : {
-			"gossi\\docblock\\" : "src"
+			"gossi\\docblock\\" : "src/",
+			"api\\": "./"
```

Comment:
this would populate into the namespace of every consuming application, I think it should be `autoload-dev` - see comment above.

Reply:
Done.